apm-sniffer/apm-sdk-plugin/thrift-plugin/src/main/java/org/apache/skywalking/apm/plugin/thrift/wrapper/ServerInProtocolWrapper.java

```diff
@@ -81,6 +90,20 @@ public TField readFieldBegin() throws TException {
             }
             return readFieldBegin();
         }
+        if (field.type == TType.STOP && !HAVE_CREATED.get()) {
```

Comment:
Would you try to do it without `ThreadLocal`?

Reply:
I don't recommend to use `ThreadLocal` here.

Reply:
@ZS-Oliver Could you explain why do you need this? Are there many wrapper instances or something?

Reply:
sorry，I took leave on Friday and didn't go to work，I will update on Monday.  happy weekend !

Reply:
I saw ServerInProtocolWrapper was initialized in TServerInterceptor.onConstruct.
The method onConstruct was called when an intercept construction method is called.
I use ThreadLocal to do the isolation, because I think this behavior will only be invoked once during the enhancement.
Are there any better way to implement this ?

Reply:
LGTM, Thanks @ZS-Oliver